x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -11,10 +11,11 @@ import { ProtectionModes } from '../types';
 /**
  * Return a new default `PolicyConfig` for platinum and above licenses
  */
-export const policyFactory = (license = ''): PolicyConfig => {
+export const policyFactory = (license = '', cloud = false): PolicyConfig => {
```

Comment:
In general, (as far as I understand the context :P ) I think the PR looks fine, good job 👍
Just to we have a small discussion I wanted to raise this: 
- why a default `false` value here? ;)